src/ploomber/tasks/notebook.py

```diff
@@ -860,10 +888,8 @@ def __init__(
         params=None,
         ext_in=None,
         static_analysis="regular",
-        local_execution=False,
```

Comment:
looks like the parameter was deleted? I thought we'd keep it (along with the default value)

Reply:
> yeah, let's make this fix independent of the local_execution parameter

What will the functionality of the `local_execution` parameter? Because earlier you mention, the local path will be default inserted into `PYTHONPATH`

Reply:
yeah, I agree there's some overlap. but local_execution is still different since if moves the current working directory.

I don't think many people use this option but it's better not to break the API unless we have a strong reason to.

Reply:
>yeah, I agree there's some overlap. but local_execution is still different since if moves the current working directory.

I am still confused. What will happen, if I set it to `True`? What should I append to the `PYTHONPATH`? Because the script local dir is already getting appended to.

```python
orig_env["PYTHONPATH"] += ':'+"what_will_be_here"
```